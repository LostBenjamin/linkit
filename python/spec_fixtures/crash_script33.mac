
[macro !]
  (((waste 4)))

[[waste INSTANCE
  12345,12345,12345
  stp
]]

