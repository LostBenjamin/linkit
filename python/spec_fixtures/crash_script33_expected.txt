Not enough remaining bytes in the last macro position for Macro 72

